fuse_viz/src/relative_pose_2d_stamped_constraint_visual.cpp

```diff
@@ -43,15 +43,29 @@
 #include <rviz/ogre_helpers/axes.h>
 #include <rviz/ogre_helpers/billboard_line.h>
 #include <rviz/ogre_helpers/movable_text.h>
+#include <tf2/utils.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 
 #include <OgreQuaternion.h>
 #include <OgreSceneManager.h>
 #include <OgreSceneNode.h>
 
+#include <algorithm>
+
```

Comment:
/home/ec2-user/workspace/test_rosdistro_fuse_PR-144/package/fuse_viz/src/relative_pose_2d_stamped_constraint_visual.cpp:64:  Add #include <string> for string  [build/include_what_you_use] [4]

```suggestion
#include <string>

```

Reply:
Fixed